spyder/locale/es/LC_MESSAGES/spyder.po

```diff
@@ -1468,29 +1475,33 @@ msgstr "Opciones avanzadas"
 
 #: spyder/plugins/completion/languageserver/confpage.py:314
 msgid "Hostname must be valid"
-msgstr ""
+msgstr "El nombre de host debe ser válido"
 
 #: spyder/plugins/completion/languageserver/confpage.py:317
 msgid ""
 "Hostname or IP address of the host on which the server is running. Must be "
 "non empty."
 msgstr ""
+"Nombre de host o dirección IP del host en el que se ejecuta el servidor. No "
+"debe estar vacío."
 
 #: spyder/plugins/completion/languageserver/confpage.py:321
 msgid "Hostname is valid"
-msgstr ""
+msgstr "El nombre del host es válido"
 
 #: spyder/plugins/completion/languageserver/confpage.py:329
 msgid "Command used to start the LSP server locally. Must be non empty"
 msgstr ""
+"Comando utilizado para iniciar el servidor LSP localmente. No debe estar "
+"vacío"
 
 #: spyder/plugins/completion/languageserver/confpage.py:336
 msgid "Program was not found on your system"
-msgstr ""
+msgstr "El programa no se encontró en su sistema"
```

Comment:
"No se encontró el programa en el sistema"?

Reply:
Much better, I'll fix it in the next commit.